content/en/docs/creating-addons/top-level-operator/addon-package.md

```diff
@@ -32,15 +39,36 @@ spec:
       targetNamespace: pko-test-ns-00-req-apy-dsy-pdy
 ```
 
-The `deadMansSnitchUrl` and the `pagerDutyKey` come from the `configMap`s with the default names in the default
-locations
-documented in
-the [addons deadMansSnitch](https://mt-sre.github.io/docs/creating-addons/monitoring/deadmanssnitch_integration/)
-and
-the [addons pagerDuty documentation](https://mt-sre.github.io/docs/creating-addons/monitoring/pagerduty_integration/)
-respectively. *To have these values inject you must maintain the default naming scheme and location of these configMaps*
-.
-
-All the values in `.spec.config.addonsv1` can be injected into the objects contained in your packageImage. See the
-[package operator documentation](https://package-operator.run/docs/guides/packaging-an-application/#go-templates) to see
-how to do this.
+* The `deadMansSnitchUrl` and `pagerDutyKey` are obtained
+from the ConfigMaps using their default names and locations.
+**IMPORTANT:** To successfully inject the `deadMansSnitchUrl` and `pagerDutyKey` values into the `ClusterPackage` manifest,
+you must keep the default naming scheme and location of the corresponding ConfigMaps.
+See the [addons deadMansSnitch](https://mt-sre.github.io/docs/creating-addons/monitoring/deadmanssnitch_integration/)
+and [addons pagerDuty](https://mt-sre.github.io/docs/creating-addons/monitoring/pagerduty_integration/)
+documentation for more information.
+
+* Additionally, all the values present in `.spec.config.addonsv1`
+ can be injected into the objects within your packageImage.
+ See the
+[package operator](https://package-operator.run/docs/guides/packaging-an-application/#go-templates)
+documentation for more information.
+
+## Tenants Onboarding Steps
```

Comment:
What is the connection between the ClusterPackage manifest and Tenant Onboarding? Put another way, how do we go from having a ClusterPackage manifest to onboarding steps?